site/content/docs/4.3/getting-started/theming.md

```diff
@@ -271,6 +271,7 @@ You can find and customize these variables for key global options in Bootstrap's
 | `$enable-caret`                              | `true` (default) or `false`        | Enables pseudo element caret on `.dropdown-toggle`. |
 | `$enable-pointer-cursor-for-buttons`         | `true` (default) or `false`        | Add "hand" cursor to non-disabled button elements. |
 | `$enable-rfs`                                | `true` (default)  or `false`       | Globally enables [RFS]({{< docsref "/getting-started/rfs" >}}). |
+| `$enable-rfs-for-spacing-utilities`          | `true` or `false` (default)        | Enables RFS for [spacing utilities]({{< docsref "/utilities/spacing" >}}). |
```

Comment:
As `$enable-rfs-for-spacing-utilities` default value is set to true, we have to adapt the docs